Description: Remove duplicate ``multi-user.target``
Author: Chloé 'kajiya' Smith <chloe.smith@canonical.com>
Origin: vendor
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/google-guest-agent/+bug/1998864
Forwarded: not-needed
Last-Updated: 2022-12-05

--- a/google-startup-scripts.service
+++ b/google-startup-scripts.service
@@ -3,7 +3,7 @@
 Wants=network-online.target rsyslog.service
 After=network-online.target rsyslog.service google-guest-agent.service
 Before=apt-daily.service
-After=cloud-final.service multi-user.target
+After=cloud-final.servicer
 Wants=cloud-final.service
 After=snapd.seeded.service
 Wants=snapd.seeded.service
